imap_processing/lo/l1a/lo_cdf_attrs.py

```diff
@@ -58,3 +64,109 @@
     units="seconds",
     label_axis="ToF",
 )
+
+### Science Counts Attributes ###
+lo_anode_rate_attrs = ScienceAttrs(
+    validmin=0,
+    validmax=GlobalConstants.INT_FILLVAL,
+    catdesc="Anode Single Rate",
+    depend_0="epoch",
+    display_type="time_series",
+    fill_val=GlobalConstants.INT_FILLVAL,
+    fieldname="Anode Single Rate",
+    format="I12",
+    var_type="data",
+    units="seconds",
+    label_axis="anode single rate",
+)
+lo_ion_rate_attrs = ScienceAttrs(
+    validmin=0,
+    validmax=GlobalConstants.INT_FILLVAL,
+    catdesc="Ion Single Rate",
```

Comment:
Should this be plural?
```suggestion
    catdesc="Ion Singles Rate",
```

Reply:
Yeah it should be